### PR TITLE
[JevpProduction] OnlTools: replace sys/types.h with stdint.h

### PR DESCRIPTION
**Pull request by veprbl**
For motivation see:
- #213
- https://github.com/star-bnl/star-sw/pull/212#discussion_r761571455

### CODE REVIEW

File: StRoot/RTS/include/rtsSystems.h [hunk 1]
```diff
@@ -1105,7 +1104,7 @@ extern inline int GET_SYSTEM(unsigned short node)
   if(id == EXT_SYSTEM) return (node & 0x03f0) >> 4;
   if(id == EXT2_SYSTEM) return (node & 0x0f80) >> 7;
   if(id == FPD_SYSTEM) {	// new FY19 extension
-	u_int sys = node & 0x0FC0 ;
+	unsigned int sys = node & 0x0FC0 ;
```
Comment (kehw) [inline StRoot/RTS/include/rtsSystems.h]: May I know the reason to treat this file differently? In all other files, u_int is replaced by uint32_t.

Comment (veprbl) [reply StRoot/RTS/include/rtsSystems.h]: Yes, I'm aware. This is just a quick test to see if I can fix CI build for the JevpProduction branch without fixing all of RTS. I have a branch for proper automated replacement for StRoot/RTS, but it was based on top of the main branch, so I would need to rebase/redo that change.

Comment (veprbl) [reply StRoot/RTS/include/rtsSystems.h]: Fixed

**Comment by akioogawa** [comment]
I think Tonko and Jeff need to review these changes and approve if ok for them, and take it to CVS as well. Otherwise code will diverge and will become a mess. I'll hold for now.

**Comment by veprbl** [comment]
Tonko, Jeff, please let me know if the change is acceptable, so that I can rebase to fix the conflicts.

**Comment by jml985** [comment]
Hi Dmitry,

I'll have to look in detail.   This is a lot of files many which run in other OS/vxworks etc...

**Comment by starsdong** [comment]
Hello All,

I am wondering the status of this PR. If this is still relevant, would appreciate if you can help review, comment/approve the request. Thanks

**Comment by jml985** [comment]
No.  Not acceptable.   Would break DAQ if back ported to the running experiment.

**Pull request closed without merging**